v2/sourcedb-to-spanner/src/main/java/com/google/cloud/teleport/v2/source/reader/io/schema/typemapping/provider/MysqlMappingProvider.java

```diff
@@ -33,7 +33,7 @@ public final class MysqlMappingProvider {
   private static final ImmutableMap<String, UnifiedTypeMapping> MAPPING =
       ImmutableMap.<String, UnifiedMappingProvider.Type>builder()
           .put("BIGINT", UnifiedMappingProvider.Type.LONG)
-          .put("BIGINT UNSIGNED", UnifiedMappingProvider.Type.DECIMAL)
+          .put("BIGINT UNSIGNED", UnifiedMappingProvider.Type.NUMBER)
```

Comment:
how do other unsigned types get handled ?

Reply:
`Integer Unsigned` gets mapped to long. 
All other smaller versions of integer like `Medium Int` `Small Int` `Tiny Int` etc. always map to `Integer` irrespective of signed or not . As the `Signed` part of the smaller types is dropped during schema mapping, we don't need separate handling for filling in Avro values.